scripts/dev_setup.sh

```diff
@@ -209,6 +210,19 @@ function install_awscli {
   aws --version
 }
 
+function install_trivy {
+  VERSION=$(trivy --version | head -1)
+  if [[ "$VERSION" != "Version: ${TRIVY_VERSION}" ]]; then
+    if [[ $(uname -s) == "Darwin" ]]; then
+      install_pkg aquasecurity/trivy/trivy brew
+    else
+      # Use trivy's script to downloads the binary based on OS and arch
+      curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b "${HOME}"/bin "v$TRIVY_VERSION"
```

Comment:
dislike this method of installation.   Most of our github installs are unzips of releases.